toolchain/check/cpp/import.cpp

```diff
@@ -2130,16 +2123,13 @@ auto ImportCppOverloadSet(
 // after overload resolution.
 static auto GetOverloadSetAccess(const clang::UnresolvedSet<4>& overload_set)
     -> SemIR::AccessKind {
-  clang::AccessSpecifier access = overload_set.begin().getAccess();
-  for (auto it = overload_set.begin() + 1; it != overload_set.end(); ++it) {
-    CARBON_CHECK(
-        (it.getAccess() == clang::AS_none) == (access == clang::AS_none),
-        "Unexpected mixture of members and non-members");
-    if (it.getAccess() < access) {
-      access = it->getAccess();
-    }
+  SemIR::AccessKind access_kind = MapAccess(overload_set.begin());
+  for (auto it = overload_set.begin() + 1;
```

Comment:
Had you considered initializing `access_kind` to `Private`, so that you wouldn't need this `begin()` offset and could only call `MapAccess` from inside the loop?

Reply:
Done.